docs/docs/cli.md

```diff
@@ -38,6 +38,10 @@ ignore patterns that omit matched files from output.
 By default, this command generates output files that omit all `state`.
 However, you can use the `--state` flag to generate output files that
 include content from a single state that you specify.
+If you would like to rename files as you copy them, use
+the `--rename` flag. The `--rename` flag takes a JSON
+string as an argument. Keys are filenames that are to be renamed, and values are the new names of those files.
```

Comment:
```suggestion
string as an argument. The JSON must represent an object whose keys are filenames that are to be renamed and values are the new names of those files.
```

Wondering if we should be explicit about the fact that specific paths would probably be ignored? i.e. this only works on _filenames_?

Reply:
Updated docs with suggestion and added sentence explicitly stating that path keys/values are not supported at this time.